File: fastmq-storage/src/main/java/com/song/fastmq/storage/BadVersionException.java
package com.song.fastmq.storage;

/**
 * Created by song on 2017/11/5.
 */
public class BadVersionException extends LedgerStorageException {

    public BadVersionException() {
    }

    public BadVersionException(String message) {
        super(message);
    }

    public BadVersionException(String message, Throwable cause) {
        super(message, cause);
    }

    public BadVersionException(Throwable cause) {
        super(cause);
    }

    public BadVersionException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace) {
        super(message, cause, enableSuppression, writableStackTrace);
    }
}

File: fastmq-storage/src/main/java/com/song/fastmq/storage/storage/support/BadVersionException.kt
package com.song.fastmq.storage.storage.support

/**
 * Created by song on 2017/11/5.
 */
class BadVersionException : LedgerStorageException {

    constructor() {}

    constructor(message: String) : super(message) {}

    constructor(message: String, cause: Throwable) : super(message, cause) {}

    constructor(cause: Throwable) : super(cause) {}

    constructor(message: String, cause: Throwable, enableSuppression: Boolean, writableStackTrace: Boolean) : super(message, cause, enableSuppression, writableStackTrace) {}
}
